PWM.py

```diff
@@ -0,0 +1,18 @@
+#code works along side GPIO.py
+#the init in GPIO.py must be called before any of these functions are called
+
+import RPi.GPIO as GPIO
+
+class signal:
```

Comment:
class names should be capitalized. especially here cuz there's also a `signal` parameter in the methods.

Reply:
actually, to follow up, none of the below methods should have a `signal` parameter...the class represents the signal, so it should just be passed to the constructor and stored as a member variable.

Reply:
ohhhhhh nvm, can you just rename `signal` to `self`. It's really confusing if you don't do that, since it's a python standard.